MyProject/cls/Table.cs

```diff
@@ -3,57 +3,92 @@
 using System.Collections.ObjectModel;
 using System.Linq;
 using System.Text;
+using System.Threading.Tasks;
+using static cls.Tbl;
 
 namespace cls
 {
     public class Tbl:ISaveRecord
     {
+        public delegate void OrderChange(string change);
+       
+        public event OrderChange OrderChanged;
+
         public int TblNumber { get; private set; }
         public int Guests { get; private set; }
 
-        private int orderNumber = SetOrderNum();
-
         public ObservableCollection<string> Order = new ObservableCollection<string>();
 
 
         public bool OrderExists { get; private set; }
 
-        public string Employee { get; private set; }
-
+        internal long _orderId { get;  set; }
 
         public Tbl(int tableNumber)
         {
             this.TblNumber = tableNumber;
-            this.Order = null;
-        }
-
-        internal static int SetOrderNum()
-        {
-            int lastOrderNum = 1; //get last order number from storage
-            return lastOrderNum + 1;
+            _orderId++;
+            OrderChanged += PutChangesToDBAsync; //is this a good idea to subscribe to event when instance of a class is created?
         }
 
         public int GetTblNum()
         {
             return this.TblNumber;
         }
 
-        public string GetOrder()
+        public ObservableCollection<string> GetOrder()
         {
-            return Order.ToList;
+            return Order;
         }
 
         public void AddOrderItem(string _item)
         {
-            
-                Order.Add(_item);
-                Save($"User {this.Employee} added record {this.Order}");
+            if (Order.Count == 1)
+            {
+                if (Order[0] == "Empty by now")
+                {
+                    Order[0] = _item;
+                    OrderChanged(_item);
+                }
+                else
+                {
+                    Order.Add(_item);
+                    OrderChanged(_item);
+                }
+            }
+            Order.Add(_item);
+            OrderChanged(_item);
+        }
+
+        public async void PutChangesToDBAsync(string _record)
```

Comment:
You know, you are the first one, who used "async void" when actually was meant to do it, as delegate signature forced you. Note: whenever you have async void, consider your method to not throw exceptions.